framework/src/auxkernels/MortarNodalAuxKernel.C

```diff
@@ -84,7 +90,12 @@ MortarNodalAuxKernelTempl<ComputeValueType>::compute()
 
   // We have to reinit the node for this variable in order to get the dof index set for the node
   _var.reinitNode();
-  _var.setNodalValue(value / total_volume);
+
+  // Allow mortar auxiliary kernels to compute quantities incrementally
+  if (!_incremental)
+    _var.setNodalValue(value / total_volume);
+  else
+    _var.setNodalValue(value / total_volume + _u_old[0]);
```

Comment:
Am I indexing `_u_old` with `[0]` correctly here (since it's always a nodal variable)?

Reply:
Yes. But you can add an assertion here that `_u_old.size() == 1` just to be sure